src/main.rs

```diff
@@ -87,9 +87,30 @@ impl std::str::FromStr for Color {
     }
 }
 
+fn get_version() -> String {
+    let pkg_version = if built_info::CI_PLATFORM.is_some() && built_info::PKG_VERSION.contains('-')
+    {
+        built_info::PKG_VERSION.replace("-dev", "-nightly")
+    } else {
+        built_info::PKG_VERSION.into()
+    };
+    let commit_hash = built_info::GIT_VERSION.and_then(|v| v.split('-').last());
+    let built_date = chrono::DateTime::parse_from_rfc2822(built_info::BUILT_TIME_UTC)
+        .unwrap()
```

Comment:
It doesn't really matter, but `built` does provide a helper to do this parsing: https://docs.rs/built/0.3.2/src/built/util.rs.html#68-72

Reply:
But using it would require adding `built` as a runtime dependency, while now it is used only as a build-time dependency.

Reply:
That is a good point! Probably not worth it.